library/std/src/Std/OpenQASM/Intrinsic.qs

```diff
@@ -22,9 +22,13 @@ export dcx, ecr, r, rzx, cs, csdg, sxdg, csx, cu1, cu3, rccx, c3sqrtx, c3x, rc3x
 
```

Comment:
It's about a function in Convert.qs, but I can't add a comment there. There's a function

```
/// The ``RESULT_AS_BOOL`` function is used to implement the cast expr in QASM for bit to bool.
/// This already exists in the Q# library, but is defined as a marker for casts from QASM.
function ResultAsBool(input : Result) : Bool {
    Std.Convert.ResultAsBool(input)
}
```

Do we still need now that we are moving everything to the standard library?

Reply:
Yes, it is a marker for now.